MantidQt/CustomInterfaces/src/Indirect/ContainerSubtraction.cpp

```diff
@@ -453,12 +441,43 @@ void ContainerSubtraction::absCorComplete(bool error) {
         "LogText", boost::lexical_cast<std::string>(m_uiForm.spShift->value()));
     m_batchAlgoRunner->addAlgorithm(shiftLog);
   }
+  // Enable post process plotting and saving
+  m_uiForm.cbPlotOutput->setEnabled(true);
+  m_uiForm.pbPlot->setEnabled(true);
+  m_uiForm.pbSave->setEnabled(true);
 
   // Run algorithm queue
   connect(m_batchAlgoRunner, SIGNAL(batchComplete(bool)), this,
           SLOT(postProcessComplete(bool)));
   m_batchAlgoRunner->executeBatchAsync();
 }
 
+/**
+ * Handles saving of workspace
+ */
+void ContainerSubtraction::saveClicked() {
+
+  // Check workspace exists
+  if (checkADSForPlotSaveWorkspace(m_pythonExportWsName, false))
+    addSaveWorkspaceToQueue(QString::fromStdString(m_pythonExportWsName));
+  m_batchAlgoRunner->executeBatchAsync();
+}
+
+/**
+ * Handles Mantid plotting of workspace
+ */
+void ContainerSubtraction::plotClicked() {
+  QString plotType = m_uiForm.cbPlotOutput->currentText();
+
+  if (checkADSForPlotSaveWorkspace(m_pythonExportWsName, true)) {
+
+    if (plotType == "Spectra" || plotType == "Both")
```

Comment:
If you see these string comparisons cropping up in several places, probably it's worth defining an enum type for the plot type. I think there's also at least two very similar "plotClicked()" methods, maybe you can share most of its code rather than duplicating? It's a small difference, but as this stuff is hard to test and is usually tested manually, having these may help prevent bugs in the plotting that can easily remain unnoticed.